src/components/Approvals/Approvals.js

```diff
@@ -332,7 +332,7 @@ function Approvals(props) {
   }
 
   function handleAction() {
-    (!comment && commentRequired) || (comment && !(/^[a-zA-Z0-9]+$/.test(comment[0]))) ? setCommentError("Enter valid reason for approving or rejecting request") : submit();
+    (!comment && commentRequired) || (comment && !(/[\p{L}\p{N}]/u.test(comment[0]))) ? setCommentError("Enter valid reason for approving or rejecting request") : submit();
```

Comment:
@maunzCache 

Thank you for the review!
Yes, I dropped `^` and `+$` intentionally.
Since the string being tested is only the first character of the comment (`comment[0]`), removing them would not affect the behavior of this function.
I thought this change would improve the code's readability.

Reply:
I might not have properly understood the original behavior then and just wanted to point this out. Will re-review this ASAP

Reply:
The change seems to be okay in comparission with the previous behavior. I guess we should get this fixed in general. Not sure why only the first character should be tested. There might be languages where one character is enough but i assume most would write a short sentence.